clicmd

```diff
@@ -83,6 +83,23 @@ function exec_tool {
   exec -a "${PNAME}" "${TOOL}" "$@"
 }
 
+# @brief log audit message
+# @param message
+function log_audit()
+{
+if [ -n "${4-}" ]; then
+  echo "$1:" "${SUDO_USER:-${USER}}:" "$(sed -e 's/\/usr\/bin\///' <<< "$2")" "$3" "$4" | systemd-cat -t CLI
+  return
+fi
+if [ -n "${3-}" ]; then
+  echo "$1:" "${SUDO_USER:-${USER}}:" "$(sed -e 's/\/usr\/bin\///' <<< "$2")" "$3" | systemd-cat -t CLI
+  return
+fi
+if [ -n "${2-}" ] || [ -n "${1-}" ]; then
```

Comment:
You're checking $2 and $1 here for being non-empty, yet don't use then in any way afterwards. Why check then?
Did you mean `-z` instead? If so, then why at the very end?
I can't get what this case is for.